### PR TITLE
Fix ordering error for calculated field handlers

### LINKED ISSUE #869: Changing the group of a user removes them from a whole meeting
I changed the group of a user from `Default` to `Admin`, but then the user is no longer participant of the groups' meeting.

See the logs:
Before:
```
backend_1           | DEBUG:openslides_backend.services.datastore.adapter:Start WRITE request to datastore with the following data: Write request: [WriteRequest(events=[{'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('meeting/1'), 'fields': {'name': 'OpenSlides - Die Veranstaltung (Teil II)', 'description': 'Presentation and assembly system', 'location': '', 'organization_tag_ids': [1], 'jitsi_domain': '', 'jitsi_room_name': '', 'jitsi_room_password': ''}}], information={FullQualifiedId('meeting/1'): ['Object updated']}, user_id=1, locked_fields={'meeting/1/committee_id': 30, 'user/1/committee_$1_management_level': 25, 'user/1/organization_management_level': 25, 'meeting/1/organization_tag_ids': 30}), WriteRequest(events=[{'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('group/2'), 'fields': {'user_ids': [1]}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('meeting/1'), 'list_fields': {'add': {'user_ids': [8]}}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('group/1'), 'fields': {'user_ids': [4, 6, 7, 8]}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('user/8'), 'fields': {'group_$_ids': ['1'], 'group_$1_ids': [1]}}], information={FullQualifiedId('group/2'): ['Object attachment to group reset'], FullQualifiedId('meeting/1'): ['Object updated'], FullQualifiedId('group/1'): ['Object attached to group'], FullQualifiedId('user/8'): ['Object updated']}, user_id=1, locked_fields={'user/1/group_$_ids': 25, 'user/1/committee_ids': 25, 'user/1/committee_$_management_level': 25, 'user/1/organization_management_level': 25, 'user/8/group_$_ids': 30, 'group/1/meeting_id': 30, 'user/8/group_$1_ids': 30, 'group/2/user_ids': 30, 'group/1/user_ids': 30, 'group/2/meeting_id': 30})]
backend_1           | DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): datastore-writer:9011
datastore-writer_1  | 2021-07-27T14:45:14.245: [/app/datastore/writer/postgresql_backend/sql_database_backend_service.py] [INFO] Created position 31
datastore-writer_1  | 2021-07-27T14:45:14.257: [/app/datastore/writer/postgresql_backend/sql_database_backend_service.py] [INFO] Created position 32
datastore-writer_1  | 2021-07-27T14:45:14.271: [/app/datastore/writer/redis_backend/redis_messaging_backend_service.py] [DEBUG] written fqfields into ModifiedFields: {"meeting/1/name": "\"OpenSlides - Die Veranstaltung (Teil II)\"", "meeting/1/description": "\"Presentation and assembly system\"", "meeting/1/location": "\"\"", "meeting/1/organization_tag_ids": "[1]", "meeting/1/jitsi_domain": "\"\"", "meeting/1/jitsi_room_name": "\"\"", "meeting/1/jitsi_room_password": "\"\"", "meeting/1/meta_position": "32", "group/2/user_ids": "[1]", "group/2/meta_position": "32", "meeting/1/user_ids": "[1, 2, 3, 4, 5, 6, 7, 8]", "group/1/user_ids": "[4, 6, 7, 8]", "group/1/meta_position": "32", "user/8/group_$_ids": "[\"1\"]", "user/8/group_$1_ids": "[1]", "user/8/meta_position": "32"}
datastore-writer_1  | 2021-07-27T14:45:14.272: [/app/datastore/writer/core/writer_service.py] [INFO] Events executed (5 UPDATE)
datastore-writer_1  | 2021-07-27T14:45:14.272: [/app/datastore/writer/core/writer_service.py] [INFO] UPDATE: ['group/1', 'user/8', 'group/2', 'meeting/1']
datastore-writer_1  | 172.21.0.12 - - [27/Jul/2021 14:45:14] "POST /internal/datastore/writer/write HTTP/1.1" 201 -
```

After:
```
backend_1           | DEBUG:openslides_backend.services.datastore.adapter:Start WRITE request to datastore with the following data: Write request: [WriteRequest(events=[{'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('group/2'), 'fields': {'user_ids': [1, 8]}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('meeting/1'), 'list_fields': {'remove': {'user_ids': [8]}}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('group/1'), 'fields': {'user_ids': [4, 6, 7]}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('user/8'), 'fields': {'number_$': ['1'], 'structure_level_$': ['1'], 'vote_weight_$': [], 'about_me_$': [], 'comment_$': [], 'vote_delegated_$_to_id': [], 'vote_delegations_$_from_ids': [], 'group_$_ids': ['1'], 'number_$1': '', 'structure_level_$1': '', 'vote_delegations_$1_from_ids': None, 'group_$1_ids': [2]}}], information={FullQualifiedId('group/2'): ['Object attached to group'], FullQualifiedId('meeting/1'): ['Object updated'], FullQualifiedId('group/1'): ['Object attachment to group reset'], FullQualifiedId('user/8'): ['Object updated']}, user_id=1, locked_fields={'user/1/group_$_ids': 25, 'user/1/committee_ids': 25, 'user/1/committee_$_management_level': 25, 'user/1/organization_management_level': 25, 'user/8/number_$': 32, 'user/8/structure_level_$': 32, 'user/8/vote_weight_$': 32, 'user/8/about_me_$': 32, 'user/8/comment_$': 32, 'user/8/vote_delegated_$_to_id': 32, 'user/8/vote_delegations_$_from_ids': 32, 'user/8/group_$_ids': 32, 'group/2/meeting_id': 32, 'user/8/vote_delegations_$1_from_ids': 32, 'user/8/group_$1_ids': 32, 'group/2/user_ids': 32, 'group/1/user_ids': 32, 'group/1/meeting_id': 32})]
backend_1           | DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): datastore-writer:9011
datastore-writer_1  | 2021-07-27T14:45:31.701: [/app/datastore/writer/postgresql_backend/sql_database_backend_service.py] [INFO] Created position 33
datastore-writer_1  | 2021-07-27T14:45:31.727: [/app/datastore/writer/redis_backend/redis_messaging_backend_service.py] [DEBUG] written fqfields into ModifiedFields: {"group/2/user_ids": "[1, 8]", "group/2/meta_position": "33", "meeting/1/user_ids": "[1, 2, 3, 4, 5, 6, 7]", "meeting/1/meta_position": "33", "group/1/user_ids": "[4, 6, 7]", "group/1/meta_position": "33", "user/8/number_$": "[\"1\"]", "user/8/structure_level_$": "[\"1\"]", "user/8/vote_weight_$": "[]", "user/8/about_me_$": "[]", "user/8/comment_$": "[]", "user/8/vote_delegated_$_to_id": "[]", "user/8/vote_delegations_$_from_ids": "[]", "user/8/group_$_ids": "[\"1\"]", "user/8/number_$1": "\"\"", "user/8/structure_level_$1": "\"\"", "user/8/group_$1_ids": "[2]", "user/8/meta_position": "33", "user/8/vote_delegations_$1_from_ids": "null"}
datastore-writer_1  | 2021-07-27T14:45:31.728: [/app/datastore/writer/core/writer_service.py] [INFO] Events executed (4 UPDATE)
datastore-writer_1  | 2021-07-27T14:45:31.728: [/app/datastore/writer/core/writer_service.py] [INFO] UPDATE: ['group/1', 'user/8', 'group/2', 'meeting/1']
datastore-writer_1  | 172.21.0.12 - - [27/Jul/2021 14:45:31] "POST /internal/datastore/writer/write HTTP/1.1" 201 -
```

As you see here `"meeting/1/user_ids": "[1, 2, 3, 4, 5, 6, 7]"` the user with id `8` is suddenly removed...

### PR DESCRIPTION
**Pull request by jsangmeister**
fixes #869

### CODE REVIEW

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: openslides_backend/action/relations/meeting_user_ids_handler.py [hunk 1]
```diff
@@ -24,30 +24,41 @@ def process_field(
...
+                new_instance = self.datastore.fetch_model(
+                    fqid, ["meeting_id"], exception=False
+                )
+                meeting_id = new_instance.get("meeting_id")
                 if not meeting_id:
                     raise ActionException(
                         f"No meeting_id found for group/{instance['id']}"
                     )
 
+        # check if removed_ids should actually be removed
```
[the remark this answers is not in the document]
Comment (jsangmeister) [reply openslides_backend/action/relations/meeting_user_ids_handler.py:49]: The handler is only called for the field `group/user_ids`, the meeting parts are a relict from the previous user structure with guest users etc. Thank you for the hint, will remove it

**Review by r-peschke — APPROVED** [review]

**Comment by jsangmeister** [comment]
The CI does not work:
```
E: Failed to fetch https://deb.nodesource.com/node_14.x/pool/main/n/nodejs/nodejs_14.17.4-1nodesource1_amd64.deb  File has unexpected size (25211968 != 25045506). Mirror sync in progress? [IP: 23.62.230.72 443]
   Hashes of expected file:
    - SHA256:796a1b5f6fa3caddd4b1f6e31bb8e98a1f289dae638a17d41ec73be0d080eceb
    - SHA1:d7e7966ad5ba47486a307d33b61b637e11c87335 [weak]
    - MD5Sum:2c4e16dcadfed0711d1a3192da4c6d1e [weak]
    - Filesize:25045506 [weak]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Service 'auth' failed to build : The command '/bin/sh -c apt-get install -y nodejs' returned a non-zero code: 100
```
Restarting did not help. Does anyone have an idea what might fix this?

**Comment by jsangmeister** [comment]
Seems like this is a general problem, in https://github.com/OpenSlides/openslides-backend/pull/861 the same error occured after restarting the jobs.

**Comment by reiterl** [comment]
I try another restart, perhaps a temp problem.

**Pull request merged**